source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,152 @@
+		history = await getFeatureHistory(featureName + '.css');
+	}
+
+	const filteredHistory = history.filter((commit: Commit) => !/^Meta|^Document|^Readme|^Lint|^Update.+dependencies/.test(commit.messageHeadline));
+	const skippedCommitsCount = history.length - filteredHistory.length;
+
+	const historyUrl = new GitHubURL(location.href);
+	historyUrl.assign({route: 'commits'});
+
+	return (
+		<details className="dropdown details-reset details-overlay d-inline-block">
+			<summary className="text-gray d-inline" aria-haspopup="true">
+				Feature history
+				<div className="dropdown-caret ml-1"/>
+			</summary>
+
+			<div className="dropdown-menu dropdown-menu-s" style={{width: 400}}>
+				<ul className="overflow-y-auto" style={{maxHeight: '60vh'}}>
+					{filteredHistory.map(commit => (
+						<li>
+							<a className="dropdown-item" href={getCommitUrl(commit)} title={commit.messageHeadline}>
+								{parseBackticks(commit.messageHeadline)}
+								<div className="text-small">
+									<relative-time datetime={commit.committedDate}/>
+								</div>
+							</a>
+						</li>
+					))}
+					{skippedCommitsCount > 0 && (
+						<>
+							<li className="dropdown-divider" role="separator"/>
+							<li>
+								<a className="dropdown-item" href={String(historyUrl)}>
+									View full history (+{skippedCommitsCount} commits)
+								</a>
+							</li>
+						</>
+					)}
+				</ul>
+			</div>
+		</details>
+	);
+}
+
+function getConversationsLink(featureName: string): Element {
+	const searchUrl = new URL('https://github.com/sindresorhus/refined-github/issues');
+	searchUrl.searchParams.set('q', `"${featureName}" sort:updated-desc`);
+
+	return <a className="ml-3" href={String(searchUrl)}>Conversations</a>;
+}
+
+async function init(): Promise<void | false> {
+	const [, currentFeature] = /features\/([^.]+)/.exec(location.pathname)!;
+	const {id, description, screenshot} = __featuresMeta__.find(feature => feature.id === currentFeature) ?? {};
+	if (!description) {
+		return false;
+	}
+
+	const descriptionElement = domify.one(description)!;
+	descriptionElement.classList.add('text-bold');
+
+	const commitInfoBox = (await elementReady('.Box-header--blue.Details'))!.parentElement!;
+	commitInfoBox.classList.add('width-fit', 'flex-auto');
+	commitInfoBox.classList.remove('mb-3');
+
+	const featureInfoBox = (
+		<div className="Box" style={{flex: '1 0 360px'}}>
+			<div className="Box-row d-flex height-full">
+				{screenshot && (
+					<a href={screenshot} className="flex-self-center">
+						<img
+							src={screenshot}
+							className="d-block border"
+							style={{
+								maxHeight: 100,
+								maxWidth: 100
+							}}/>
+					</a>
+				)}
+				<div className="ml-3 flex-auto">
+					{descriptionElement}
+					{await getHistoryDropdown(id!)}
+					{getConversationsLink(id!)}
+				</div>
+			</div>
+		</div>
+	);
+
+	wrapAll([commitInfoBox, featureInfoBox], <div className="d-flex flex-wrap" style={{gap: 16}}/>);
+}
```

Comment:
Instead of having the whole thing `await` the API call, can you just append the dropdown later?


Example here:
```suggestion
	select('.rgh-feature-box').append(await getHistoryDropdown(id))
}
```